applications/nrf5340_audio/dfu/conf/overlay-dfu_external_flash.overlay

```diff
@@ -36,6 +30,12 @@
 		t-exit-dpd = <5000>;
 		dpd-wakeup-sequence = <30000 20 45000>;
 	};
+	sdhc0: sdhc@1 {
```

Comment:
Nice catch. This was changed upstream.

Reply:
But why do we need to have this here? THis has nothing to do with ext flash?

Reply:
This change is needed as the external flash previously was listed as unit #2 on the bus, while this is now occupied by the HW codec. The change upstream was to reserve the #0 slot for an Arduino shield connection, which the external flash shield is. This change only relates to the order of the cs-pins in the DTS file.